src/corner_kick/src/style/Theme.tsx

```diff
@@ -10,9 +10,28 @@ import { theme } from 'SRC/constants';
 import { createGlobalStyle, ThemeProvider } from 'SRC/utils/styled-components';
 
 const GlobalStylesheet = createGlobalStyle`
+    @import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+    * {
+        box-sizing: border-box;
+    }
+
+
     body {
+        margin: 0;
+
+        overflow: hidden;
+        user-select: none;
+
         background: ${(props) => props.theme.colors.bg};
         color: ${(props) => props.theme.colors.fg};
+
+        font-family: 'Open Sans', sans-serif;
+        font-size: 14px;
+    }
+
+    a {
+        text-decoration: none;
     }
```

Comment:
These changes seem out-of-scope? They're small enough that I don't mind if they are, but I just want to confirm what they're for?

Reply:
Storybook struggled with importing .css files, so I merged the content from base.css to Theme.tsx

Reply:
Storybook struggled with importing .css files, so I merged the content from base.css to Theme.tsx